CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # React Native Module Changelog
 
+## Version 13.0.1 - November 5, 2021
+Patch release that fixes preferences resetting when upgrading to SDK 15/16. This update will restore old preferences that have not been modified in the new SDK version.
```

Comment:
see my flutter example for PR. i think it should be in the context of react plugin versions

Reply:
Good call. This is updated 👍